src/v3/test/integration/authenticator-verification-email.test.tsx

```diff
@@ -20,7 +19,12 @@ import sessionExpiredResponse from '../../src/mocks/response/idp/idx/identify/er
 
 describe('Email authenticator verification when email magic link = undefined', () => {
   describe('renders correct form', () => {
+    let mockSystemTime: number;
+
     beforeEach(() => {
+      // Mock system time for triggering resend email reminder element
+      mockSystemTime = Date.now();
+      Date.now = jest.fn(() => mockSystemTime);
```

Comment:
If you replace the global Date.now value/implementation, it should be reset. 

Another option is to instead use `spyOn` so you dont have to reset the original implementation. 

i.e.
```
const getNow = () => new Date(Date.now());

test('It should create correct now Date', () => {
  jest
    .spyOn(global.Date, 'now')
    .mockImplementationOnce(() =>
      1234567889
    );

  expect(getNow()).toEqual(new Date('2019-05-14T11:01:58.135Z'));
});
```

See: https://codewithhugo.com/mocking-the-current-date-in-jest-tests/